app/containers/MessageBox/RecordAudio.js

```diff
@@ -64,6 +67,15 @@ export default class RecordAudio extends React.PureComponent {
 		};
 	}
 
+	componentDidMount() {
+		const { recordStartState, toggleRecordAudioWithState } = this.props;
+
+		if (recordStartState) {
```

Comment:
Could there be a scenario where is mounts with `recordStartState` being false, but then the prop later changes to true? If so, would `this.startRecordingAudio();` also need to be called somewhere else as well?

Reply:
It was wrong because the changes to keep the keyboard while composers are switching, but that has already been resolved.